### PR TITLE
Geek Girls Carrots added

### PR DESCRIPTION
**Pull request by wiktoriatomzik**
I've added the organization I volunteer for - Geek Girls Carrots.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,6 +153,10 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     The Women in Tech Summit inspires, educates and connects women in the technology industry.
 
+  - [Geek Girls Carrots](http://gocarrots.org) – *Loc:* <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e9/Flag_of_Poland_%28normative%29.svg/1280px-Flag_of_Poland_%28normative%29.svg.png" alt="Poland" width="30">
+
+    Geek Girls Carrots is a global grassroots organisation focused on women in tech and female entrepreneurs. We gather to inspire, learn from each other, share knowledge. We believe that girls in new technologies are the avant-garde of innovation. We want to overcome stereotypes by bringing more diversity into tech world. Find out what we do, where and when you can join and start coding!
```
Comment (harsh183) [inline README.md]: Well this page doesn't represent them so we can't use the word `we`. Other than that this looks pretty okay so update that and this should be good enough to merge.

**Comment by wiktoriatomzik** [comment]
How about this version? :)

**Comment by harsh183** [comment]
Looks good. Merging.

@cristianoliveira Take a look as well. Looks fine to me.

**Comment by harsh183** [comment]
Looks good. Merging.

@cristianoliveira Take a look as well. Looks fine to me.

**Comment by cristianoliveira** [comment]
Hey @harsh183 Thanks for taking care of it. I was way too busy with this GDPR thing 😫 
I trust on your judgment :)

@wiktoriatomzik thanks for your contributions!

**Comment by wiktoriatomzik** [comment]
Pleasure is all mine :)

pt., 25 maj 2018 o 10:54 Cristian Oliveira <notifications@github.com>
napisał(a):

**Pull request merged**